### ISSUE TITLE
Parallel Scene builds

### ISSUE THREAD

**Issue by himanshm** [ManimCommunity/manim#4942]
## Description of proposed feature
The scenes are split `with self.voiceover("Something here") as tracker:` blocks, there should be an option to generate these frames in parallel.

## How can the new feature be used?
This features will lead to better speed for builds

## Additional comments
- This doesn't need to be the case when you choose the Recorder option from manim_voiceover
- But, for other voiceover options, you should offer parallel build and keep a file in system that keeps track of where things are, with data like position of objects and what they are, if they are present and animation on them is applied in a parallel build proceed, otherwise warn the user.
